cpp/data/rss_email.template.en

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+    <body>
+Dear {user_name},
+
+The following items from your selected RSS feeds are new:</br>
```

Comment:
Couldn't test it due to the other SEVERE error yet, but the standalone </br> also looks wrong.

Reply:
Besides... the other newlines on line 4 and 5 normally won't have any effect, so it might be better if you do something like this:

<p>Dear {user_name},</p>
<p>The following items from your selected RSS feeds are new:</p>

Reply:
Done.